compiler/test/suites/chars.re

```diff
@@ -67,10 +67,16 @@ describe("chars", ({test, testSkip}) => {
   assertRun("char_toString_escape8", {|print(('\v', 1))|}, "('\\v', 1)\n");
   assertRun("char_toString_escape9", {|print(('\'', 1))|}, "('\\'', 1)\n");
   assertCompileError(
-    "char_illegal",
+    "char_illegal1",
     "'abc'",
     "This character literal contains multiple characters: 'abc'\nDid you mean to create the string \"abc\" instead?",
   );
+  assertCompileError(
+    "char_illegal2",
+    {|'{"test": 1}'|},
+    {|This character literal contains multiple characters: '\{"test": 1\}'
+Did you mean to create the string "\{\\"test\\": 1\}" instead?|},
```

Comment:
Do we really need to escape `{` and `\`?

Reply:
Alex had noted the rest suite uses regex. And offered that solution and it worked so I took it. Everything else I tried didn't work so I think maybe.

Reply:
Ah, I see.